app/models/user.rb

```diff
@@ -36,6 +36,8 @@ class User < ApplicationRecord
   validates :reset_digest, uniqueness: true, if: :reset_digest?
   validates :activation_digest, uniqueness: true, if: :activation_digest?
 
```

Comment:
Shouldn't we add in model validation for uniqueness?
The database exceptions aren't the same as the `ActiveRecord` ones.